### DISCUSSION TITLE
System Wide Notification

### DISCUSSION THREAD

**Discussion by avinashkanaujiya** [TriliumNext/Trilium#1360] [Q&A]
How to write a script within trilium to push a notification system wide? or is it just not possible?
I'm a little new to programming so a push in right direction will be helpful.

**Comment by zadam** [comment]
For system wide notifications you would need to use electron APIs. I think it should be possible to do that but I didn't try myself.

**Reply by avinashkanaujiya** [reply]
Thanks! I will try that.
